docs/conf.py

```diff
@@ -295,3 +295,5 @@
 # ------------------------------------
 
 hoverxref_auto_ref = True
+hoverxref_project = "scrapy"
```

Comment:
Apparently this is necessary now, otherwise we get
```
Warning, treated as error:
hoverxref extension is not fully configured. Set hoverxref_project and hoverxref_version in your conf.py file.
```
(https://travis-ci.org/github/scrapy/scrapy/jobs/673517936#L285-L286)